docker/scripts/clean.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Run this shell script to rm existing CL Docker images and pull the latest, freshest ones.
+# Use when:
+## 1) Merging `main` into a working branch;
+## 2) Forking off of `main` to start a new working branch
+# Run it before using the build script. Usage of both of these shell scripts is optional.
+
+# Mostly from here: https://stackoverflow.com/a/39127792/3256077
+
+# stop all running CL containers
+docker-compose -f ../courtlistener/docker-compose.yml stop
+# rm said containers
+docker-compose -f ../courtlistener/docker-compose.yml rm --force
+# rm the locally-stored CL images themselves
+docker rmi $(docker-compose -f ../courtlistener/docker-compose.yml config | awk '/freelawproject/{if ($1 == "image:") print $2;}')
```

Comment:
My understanding is that it's worth removing the images largely because if we're using the `:latest` tag for a lot of stuff, what is labeled as such locally may not match what is up there in the registry. I think some might believe as a matter of like philosophy that it's better to totally delete these things and then pull them fresh than to trust the `pull` command to update them even though it's supposed to do that. 

This was also being done in Chris's original script (via a different set of commands).